### PR TITLE
add jsPlumb@2.2.8 and update its package.json

### PR DESCRIPTION
**Pull request by kennynaoh**
Pull request for issue: #10030 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.
@pvnr0082t @x09326 Please help me review this PR, thanks!
Manually add old versions from git repo because I remove the version pattern of file name and the file structure is different before v1.4.1
Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jsplumb/jsPlumb
 * Official website (optional, not the repository): https://jsplumbtoolkit.com/
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3153
   - Count of watchers: 212
   - Count of forks: 809
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by kennynaoh** [comment]
Sorry, I think I need to ask author to remove the version pattern of file name in the latest release.

**Comment by kennynaoh** [comment]
@x09326 @pvnr0082t Please help me review this PR. Thanks!

**Comment by kennynaoh** [comment]
@pvnr0082t @x09326 I have fixed the commit message. Please help me review it. Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

File: ajax/libs/jsPlumb/package.json [hunk 1]
```diff
@@ -15,11 +15,23 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/sporritt/jsPlumb.git"
+    "url": "https://github.com/jsplumb/bower-jsplumb.git"
```
Comment (PeterDaveHello) [inline ajax/libs/jsPlumb/package.json]: You should not change it ...

**Comment by kennynaoh** [comment]
@x09326 @pvnr0082t I have fixed the repository. Please help me review it. Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by kennynaoh** [comment]
I am not sure how to wrap the commit message, could you please help me? Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh Simple press your enter key.

**Comment by PeterDaveHello** [comment]
For example ...

```
Manually add old versions from https://github.com/jsplumb/jsPlumb because the file name with
version pattern and the file structure is different before v1.4.1
```
->
```
Manually add old versions from https://github.com/jsplumb/jsPlumb                                                                                                                             
because the file name with version pattern and the file structure is                                                                                                                          
different before v1.4.1
```

**Comment by kennynaoh** [comment]
@PeterDaveHello I understood.

**Comment by kennynaoh** [comment]
@pvnr0082t @x09326 I have fixed the commit message. Please help me review it again, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
> Manually change the file name structure and add git auto-update config to update the latest release automatically

Filename changes are never be auto, so there is no need to add *Manually*.

Repository info still out-dated, please update in the first commit, or add another commit in this PR to update it.

**Comment by kennynaoh** [comment]
@pvnr0082t @x09326 I have fixed the commit message and update the git repo, please help review it. Thanks!

**Comment by kennynaoh** [comment]
@pvnr0082t @x09326 I fixed the wrong commit. I have updated now. Please help me review it again. Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
No library name mentioned in the third commit 0366bb9 ?

**Comment by kennynaoh** [comment]
@pvnr0082t @x09326 I have updated the commit message. Please help me review it, thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh please use a more precise title next time.

**Comment by kennynaoh** [comment]
OK

**Pull request merged**